render/mapping.go

```diff
@@ -441,8 +441,10 @@ func MapProcess2Container(n RenderableNode, _ report.Networks) RenderableNodes {
 		id = MakeContainerID(containerID)
 		node = NewDerivedNode(id, n)
 	} else {
+		nCopy := n.Copy()
+		nCopy.Node = nCopy.Node.WithID("").WithTopology("") // Wipe the ID so it cannot be rendered.
```

Comment:
The LGTM

On Friday, February 12, 2016, Paul Bellamy <notifications@github.com> wrote:

> In render/mapping.go
> <https://github.com/weaveworks/scope/pull/956#discussion_r52740480>:
>
> > @@ -441,8 +441,10 @@ func MapProcess2Container(n RenderableNode, _ report.Networks) RenderableNodes {
> >  		id = MakeContainerID(containerID)
> >  		node = NewDerivedNode(id, n)
> >  	} else {
> > +		nCopy := n.Copy()
> > +		nCopy.Node = nCopy.Node.WithID("").WithTopology("") // Wipe the ID so it cannot be rendered.
>
> It removes the red section.
> [image: screen shot 2016-02-12 at 13 28 35]
> <https://cloud.githubusercontent.com/assets/250199/13008184/cba26546-d18c-11e5-8846-c429505920ae.png>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/weaveworks/scope/pull/956/files#r52740480>.
>